pkg/addons/embeddedclusteroperator/embeddedclusteroperator.go

```diff
@@ -40,7 +40,11 @@ var (
 	rawmetadata []byte
 	// Metadata is the unmarshal version of rawmetadata.
 	Metadata release.AddonMetadata
-	// Overwritten by -ldflags in Makefile
+)
+
+// Overwritten by -ldflags in Makefile
+var (
+	UtilsImage                    = "busybox:latest"
```

Comment:
why does this sometimes come from the makefile and sometimes from the metadata.yaml file?

Reply:
Im just trying to unrevert this. ultimately we should move this to a makefile but i will follow up with that.

Reply:
i will add a metadata in a follow up when i add image gcr.io/kubebuilder/kube-rbac-proxy